sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_apache_avro_encoder.py

```diff
@@ -73,9 +73,9 @@ def encode(
 
     def decode(
         self,
-        content,  # type: Union[bytes, BinaryIO]
-        reader,  # type: DatumReader
-    ) -> ObjectType:
+        content: Union[bytes, BinaryIO],
+        reader: DatumReader,
+    ) -> Dict[str, Any]:
         """Read the binary representation into a specific type.
```

Comment:
same as the other decode  (content, reader) as signature but docstring is content, schema, readers_schema